shoop/xtheme/view_config.py

```diff
@@ -31,9 +32,15 @@ def __init__(self, theme, view_name, draft):
         :type view_name: str
         :param draft: Load in draft mode?
         :type draft: bool
+        :param global_type: Boolean indicating whether this is a global config
+        :type global_type: bool|False
         """
         self.theme = theme
-        self.view_name = force_text(view_name)
+        self.global_type = bool(global_type)
+        if not global_type:
+            self.view_name = force_text(view_name)
+        else:
+            self.view_name = XTHEME_GLOBAL_VIEW_NAME
```

Comment:
Is this even necessary since you are already overring the view_name when calling this. Or just remove the view_name override in rendering.py and editor.py since you are still doing it here.